static/css/styles.css

```diff
@@ -583,6 +583,24 @@ h2 {
 */
 
 
+.other-products-heading {
+  margin: 40px 10px 10px 10px;
+  width: 94%;
+  font-weight: bold;
+  color: #55565a;
+  font-size: 18px;
+  padding-bottom: 10px;
+  border-bottom: 1px solid black;
+}
+
+.docset-link span {
+  color: black;
+}
```

Comment:
What is this needed for? I can't see any different with it toggled off.

```suggestion
```

Reply:
I was mostly duplicating the manual styling and I guess I didn't really check if it changed anything or not.